### PR TITLE
[DOC] Improve project description and README

### LINKED ISSUE #2: [DOC] Improve project description and README
Current baseline: `A R package which embeds BPMN Visualization`

It doesn't describe the purpose of the package. See also https://github.com/process-analytics/bpmn-visualization-R/pull/1#discussion_r693962789.
We will keep the ref to the `bpmn-visualization-js` integration.

**Sources of inspiration**
- https://github.com/rich-iannone/DiagrammeR
- https://github.com/dmurdoch/rgl/
- for DESCRIPTION, see the list of packages available on CRAN: https://cran.r-project.org/web/packages/available_packages_by_name.html#available-packages-G

**Tasks**
- [x] repository description: _A R package for visualizing process execution data on BPMN diagrams_ 
- [x] R DESCRIPTION file. Remove ref to _R package_, for sure, this is this a R package 😙 #39 
- [x] readme #39

### PR DESCRIPTION
**Pull request by csouchet**
Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,8 @@
 </div>  
 <br>
 
-This project is an R interface to the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library.
+This R package is an interface to the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library (TypeScript). \
+It enables to visualize the execution data of the processes on BPMN diagrams, using overlays, style customization and interactions.
```
Comment (tbouffard) [inline README.md:25]: Using something closer to the repository description

```suggestion
A R package for visualizing process execution data on BPMN diagrams, using overlays, style customization and interactions.

It is an interface to the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) TypeScript library binds to R thanks to [htmlwidgets](http://www.htmlwidgets.org/).
```

File: README.md [hunk 2]
```diff
@@ -61,6 +54,13 @@ Or
 bpmn_file <- "<?xml version="1.0" encoding="UTF-8"?> ...xml_content"
 ```
 
+💡 The package provides some [BPMN examples](./inst/examples). \
```
Comment (tbouffard) [inline README.md:57]: nice ❤️

File: DESCRIPTION [hunk 3]
```diff
@@ -1,10 +1,10 @@
 Package: bpmnVisualization
 Type: Package
-Title: BPMN Visualization - R Package
+Title: BPMN Visualization for R
```
Comment (tbouffard) [inline DESCRIPTION:3]: Possible titles
- Display BPMN Diagrams and execution data

Comment (csouchet) [inline DESCRIPTION:3]:
```suggestion
Title: Display BPMN diagram with execution data
```

File: README.md (hunk 1 above)
Comment (csouchet) [inline README.md:25]:
```suggestion
This R package enables to visualize the execution data of the processes on BPMN diagrams, using overlays, style customization and interactions, with [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library (TypeScript).
```

Comment (csouchet) [reply README.md:25]:
```suggestion
This R package enables to visualize the execution data of the processes on BPMN diagrams, using overlays, style customization and interactions, with [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library (TypeScript). \
Using JavaScript in R is handled by [htmlwidgets](http://www.htmlwidgets.org/) framework.
```

File: DESCRIPTION (hunk 3 above)
Comment (csouchet) [reply DESCRIPTION:3]:
```suggestion
Title: Display a process on BPMN diagram with execution data
```

File: README.md (hunk 1 above)
Comment (tbouffard) [reply README.md:26]: Final proposal agreed with @csouchet 

```suggestion
This project is an R package for visualizing process execution data on BPMN diagrams, using overlays, style customization and interactions.

It is made possible by [htmlwidgets](http://www.htmlwidgets.org/), which provides an easy-to-use framework for bringing together R and the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) TypeScript library.
```

File: DESCRIPTION (hunk 3 above)
Comment (tbouffard) [reply DESCRIPTION:3]: Final proposal agreed with @csouchet

```suggestion
Title: Visualize process execution data on BPMN diagrams
```

**Review by tbouffard — APPROVED** [review]

**Pull request merged**